backend/mora/service/address.py

```diff
@@ -415,11 +495,13 @@ def convert(effect):
                 addr[mapping.VALIDITY] = util.get_effect_validity(addrrel)
 
                 if self.scope.path == 'organisation/bruger':
+                    from . import employee
```

Comment:
Erm, local imports are both a) slow and b) ugly. I'd rather see this in another module than `common` than a local import 😕

Reply:
It was purely to resolve an iffy circular dependency. I'll try to move stuff around to avoid this.